src/Benchmarks/FortunesEfMiddleware.cs

```diff
@@ -44,9 +44,7 @@ public FortunesEfMiddleware(RequestDelegate next)
 
         private static async Task<IEnumerable<Fortune>> LoadRows(ApplicationDbContext dbContext)
         {
-            var result = new List<Fortune>();
-
-            result.AddRange(await dbContext.Fortunue.ToListAsync());
+            var result = await dbContext.Fortune.ToListAsync();
```

Comment:
Here you go: https://github.com/Reactive-Extensions/Rx.NET/blob/master/Ix.NET/Source/System.Interactive.Async/AsyncEnumerable.Aggregates.cs#L613